src/utils/fhirUtils.ts

```diff
@@ -61,17 +61,25 @@ export function getHumanName(person: fhir.HumanName | fhir.HumanName[]): string
 }
 
 export function createDocumentReference(
-  selected: string,
+  data: string,
+  labelOrCondition: string,
   patient: fhir.Patient
 ): fhir.DocumentReference {
   return {
     resourceType: 'DocumentReference',
+    id: btoa(labelOrCondition + Math.floor(Math.random() * 1000)),
```

Comment:
This PR pulls in the `uuid` dependency for creating ids and it is used here: https://github.com/mcode/pathways/blob/6cc57a5e50e460f47c978d53a7601ea5ad634f23/src/utils/fhirUtils.ts#L12.

Any reason for not using that library here?  Seems like we should if we have it.

Reply:
I think this was already updated

Reply:
hmm, I don't see it updated...

Reply:
Have you pulled the latest? It should be using uuid to generate the id now.